### PR TITLE
feat: Hypixel motion disabler + lowhop

### PR DESCRIPTION
**Pull request by localpthebest**

### CODE REVIEW

**Comment by localpthebest** [comment]
Darn detekt!

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/disabler/disablers/DisablerHypixel.kt [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import net.ccbluex.liquidbounce.event.events.*
+import net.ccbluex.liquidbounce.event.handler
+import net.ccbluex.liquidbounce.features.module.modules.exploit.disabler.ModuleDisabler
+import net.minecraft.network.packet.s2c.play.*
+
+internal object DisablerHypixel : ToggleableConfigurable(ModuleDisabler, "Hypixel", false) {
+    var ticks = 0
+    var airticks = 0
+    var disabled = false
+    var jump = false
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/disabler/disablers/DisablerHypixel.kt]: airTicks

also, make them private, same for all the handlers below.

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/disabler/disablers/DisablerHypixel.kt [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        if(!player.isOnGround) {
+            airticks++
+        } else {
+            airticks = 0
+        }
+        if(player.isOnGround && jump) {
+            player.jump()
+            jump = false
+            disabled = true
+        } else if(disabled && airticks >= 10) {
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/disabler/disablers/DisablerHypixel.kt]: inconsistent formatting

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/disabler/disablers/DisablerHypixel.kt [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                disabled = false
+                ticks = 0
+            }
+        }
+    }
+
+    @Suppress("unused")
+    private val onNetworkTick = handler<PlayerNetworkMovementTickEvent> { event ->
+        if(event.state != EventState.PRE) return@handler
+        if(!player.isOnGround) {
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/disabler/disablers/DisablerHypixel.kt]: inconsistent formatting

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/disabler/disablers/DisablerHypixel.kt [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+            player.jump()
+            jump = false
+            disabled = true
+        } else if(disabled && airticks >= 10) {
+            if(airticks % 2 == 0) {
+                event.x += 0.095
+            }
+            player.velocity.x = 0.0
+            player.velocity.y = 0.0
+            player.velocity.z = 0.0
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/exploit/disabler/disablers/DisablerHypixel.kt]: use setVelocity method instead

**Review by 1zun4 — APPROVED** [review]

**Pull request merged**